common/include/Pcsx2Defs.h

```diff
@@ -168,7 +168,7 @@ static const int __pagesize = PCSX2_PAGESIZE;
 #define PCSX2_ALIGNED16_EXTERN(x) extern __declspec(align(16)) x
 
 #define __noinline __declspec(noinline)
-#define __threadlocal __declspec(thread)
+#define thread_local __declspec(thread)
```

Comment:
You shouldn't redefine a keyword unless there's a need to. You should have just removed the line.